bin/micro.js

```diff
@@ -2,48 +2,50 @@
 
 // Native
 const path = require('path')
+const { existsSync } = require('fs')
 
 // Packages
-const updateNotifier = require('update-notifier')
-const nodeVersion = require('node-version')
-const args = require('args')
-const isAsyncSupported = require('is-async-supported')
-
-// Ours
-const pkg = require('../package')
-
-// Throw an error if node version is too low
-if (nodeVersion.major < 6) {
-  console.error(
-    `Error! Micro requires at least version 6 of Node. Please upgrade!`
-  )
-  process.exit(1)
-}
+const parseArgs = require('mri')
 
-// Let user know if there's an update
-// This isn't important when deployed to Now
-if (!process.env.NOW && pkg.dist) {
-  updateNotifier({ pkg }).notify()
-}
+// Utilities
+const serve = require('../lib')
+const handle = require('../lib/handler')
+const generateHelp = require('../lib/help')
+const { version } = require('../package')
+const logError = require('../lib/error')
 
-args
-  .option('port', 'Port to listen on', parseInt(process.env.PORT, 10) || 3000, Number)
-  .option(['H', 'host'], 'Host to listen on', '0.0.0.0')
-  .option(['s', 'silent'], 'Silent mode')
-
-const flags = args.parse(process.argv, {
-  minimist: {
-    alias: {
-      p: 'port',
-      H: 'host',
-      s: 'silent'
-    },
-    boolean: ['silent'],
-    string: ['host']
+// Check if the user defined any options
+const flags = parseArgs(process.argv.slice(2), {
+  string: ['host', 'port'],
```

Comment:
You can set `default`s for these two & the type will be inferred; aka:

```js
default: {
  host: 'localhost',
  port: 3000
}
```

Will also apply the defaults & type-casting to their aliases.